bcp/client/src/webpack/pages/List.js

```diff
@@ -1,16 +1,174 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import useStores from "../hooks/use-stores";
 import {observer} from "mobx-react";
+import Table from 'react-bootstrap/Table';
+import {fetchAssetList} from "../helper/toGeoJSON";
+import {getLocationCell} from '../helper/map-dashboard';
+import {ASSET_MODE, DEBOUNCE_DELAY_MS, SORTKEYID} from '../constant';
+import {filterAssets} from "../helper/map-math";
+import _ from 'lodash';
```

Comment:
As a general rule, it's best to import specific features where possible. This is especially true for massive libs like lodash.
`import debounce from 'lodash.debounce';`
In package.json
`"lodash.debounce": "4.0.8",`